packages/i18n/src/test/index.js

```diff
@@ -84,4 +88,21 @@ describe( 'i18n', () => {
 			expect( result ).toBe( 'bonjour Riad' );
 		} );
 	} );
+
+	describe( 'setAdditionalLocale' , () => {
```

Comment:
Code style: No space before comma ([reference](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls))